utils/custom_tags.py

```diff
@@ -0,0 +1,13 @@
+
+from django import template
+
+#  Initialize tags
+register = template.Library()
+
+
+@register.filter(name='dict_lookup')
+def dict_lookup(dictionary, key):
+    """
+    Retrives value from dict
+    """
+    return dictionary.get(key, "")
```

Comment:
Do we need this?
https://docs.djangoproject.com/en/4.0/ref/templates/api/#variables-and-lookups

Reply:
@thenav56  In `Template` I din't see template_path arg,  but in `render_to_string` we chan pass template_path easily.

Reply:
Oh no.
Do we need to use custom filter for dictionary value lookup? Can't we use this https://docs.djangoproject.com/en/4.0/ref/templates/api/#variables-and-lookups?